src/org/etools/j1939_84/bus/j1939/packets/model/SpnDefinition.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2020. Equipment & Tool Institute
+ */
+package org.etools.j1939_84.bus.j1939.packets.model;
+
+public class SpnDefinition {
+    public int bitLength;
+    public int slotNumber;
+    public int spnId;
+    public int startBit;
+    public int startByte;
+}
```

Comment:
What about SPN label?  Isn't spn bit length redundant with the slot information?

Reply:
The SPN Label is already available via `Lookup.getSpnName()` 
`bitLength` here is redundant.  I can use the value from the Slot instead.

Reply:
Because the J1939DA extract fr the PGN->spn->slot is redundant with the one used in Lookup, I'll replace that logic.  I'll also update the  other csv files referenced in lookup with new extracts from J1939DA 2020.  It makes more sense to add the spn label to the spndefinition.

Reply:
I can add the label to the definition.  However, there are several other places where we use Lookup to find the label for the SPN.  I'll use the label on the definition in this case, and you can fix the other places.